### DISCUSSION TITLE
Wiki Page Update Notification

### DISCUSSION THREAD

**Discussion by Nonperson1960** [mriscoc/Ender3V2S1#650] [General]
First, I apologize if this isn't the correct place to post this. Feel free to delete it if neccessary.

As a documentation nerd, it would be a great feature if I could get a notification if a wiki page was updated. I didn't see anything in github notification docs so I thought I'd ask the group.
Why? Sometimes it's easier for me to print out instructions, make notes as I make changes and have a record of when and what was changed. YouTube videos are excellent but pausing, replaying sections to better understand the content is sometimes harder than just reviewing a manual page.
So, I saved each wiki page as a pdf on my PC. I can then print out or view the pdf so if I need to check a setting or make a tweak I can refer back to it.
With each release, I need to then check the wiki to see if a page has been updated. I then update the pdf file on my PC.

The firmware is wonderful, thank you Miguel for your generousity to the community.

**Comment by mriscoc** [comment]
Hi, thank you for your kind words.
I didn't find any settings to enable wiki update notification.
